packages/@okta/vuepress-site/docs/reference/api/factor-admin/index.md

```diff
@@ -301,6 +303,8 @@ curl -v -H "Authorization: SSWS yourtoken" \
 
 ### Deactivate SMS
 
+<ApiLifecycle access="ie" /><br>
```

Comment:
We are using this label for things that are only available as a part of OIE. This seems to be the opposite case?

Reply:
I misunderstood then, I thought the label was being used to highlight any OIE-specific behavior changes, not just additional features.  Is there any guidance on how how to document these kinds of limitations?